### ISSUE TITLE
Formula with query including average session duration fails

### ISSUE THREAD

**Issue by benjackwhite** [PostHog/posthog#15090]
## Bug description

There seems to be an issue with Formulas when one of the values uses Session Duration as an averaged property value

## How to reproduce

1. Go to this insight (Posthog employees) https://app.posthog.com/insights/0OZy62ds
2. Try to add the formula `A/B` (or actually even just `A`)
3. Crash!

Related Sentry issue (I believe) https://posthog.sentry.io/issues/3840456243/

## Environment

- [x] PostHog Cloud US, project ID: [please provide from https://app.posthog.com/project/settings#project-id]
- [ ] PostHog Cloud EU, project ID: [please provide from https://eu.posthog.com/project/settings#project-id]
- [ ] PostHog Hobby self-hosted with `docker compose`, version/commit: [please provide]
- [ ] PostHog self-hosted with Kubernetes (deprecated, see ["Sunsetting Kubernetes support"](https://posthog.com/blog/sunsetting-helm-support-posthog)), version/commit: [please provide]

**Comment by Twixes** [comment]
+1 based on ZEN-2699

**Comment by Twixes** [comment]
+1 based on ZEN-2854

**Comment by Twixes** [comment]
+1 based on ZEN-3329

**Comment by thmsobrmlr** [comment]
Are we sure this is fixed? The mentioned insight still doesn't work. Don't know what the original error was, but here's a Sentry link of the current one: https://posthog.sentry.io/issues/4228732607/events/714b21e38bd8435c80e5d354b8b9f752/

**Comment by Twixes** [comment]
That is true, I think filtering by session duration was fixed, but not averaging it.

**Comment by Twixes** [comment]
All of these cases appear to work now, so closing. If anything else comes up around the session duration + formula combination that I missed, feel free to reopen!

**Comment by thmsobrmlr** [comment]
The following formula doesn't seem to work @Twixes  (originating from this support request https://posthoghelp.zendesk.com/agent/tickets/4501 (moved to PostHog: https://us.posthog.com/project/2/support/tickets/6978)). Associated sentry issues is here https://posthog.sentry.io/issues/4228732607/. Do you have an idea why this happens and can have a look? Otherwise let me know and I'll add it to my todos.

---

<img width="2277" alt="Screenshot 2023-07-25 at 13 52 28" src="https://github.com/PostHog/posthog/assets/1851359/6327b83e-0ade-4709-add8-f39399311d32">


```json
{
  "kind": "InsightVizNode",
  "source": {
    "kind": "TrendsQuery",
    "filterTestAccounts": true,
    "series": [
      {
        "kind": "EventsNode",
        "event": "$pageview",
        "name": "$pageview",
        "math": "sum",
        "math_property": "$session_duration"
      },
      {
        "kind": "EventsNode",
        "event": "$pageview",
        "name": "$pageview",
        "math": "dau"
      }
    ],
    "interval": "day",
    "trendsFilter": {
      "display": "ActionsLineGraph",
      "formula": "A/B"
    }
  },
  "full": true
}
```
